lib/handlers/puback.js

```diff
@@ -3,6 +3,7 @@
 function handlePuback (client, packet, done) {
   var persistence = client.broker.persistence
   persistence.outgoingClearMessageId(client, packet, done)
+  client.broker.emit('deliver', packet, client)
```

Comment:
I believe `delivered` or `ack` could be a better name, in Mosca we are using `delivered` @mcollina